tests/Features/ChildModelFillablesMergeWithParentModelFillablesTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Parental\Tests\Features;
+
+use Parental\Tests\Models\Event;
+use Parental\Tests\Models\Workshop;
+use Parental\Tests\TestCase;
+
+class ChildModelFillablesMergeWithParentModelFillablesTest extends TestCase
+{
+    /** @test */
+    function child_fillables_are_merged_with_parent_fillables()
+    {
+        $workshop = Workshop::create([
+            'name' => 'Scaling Laravel',
+            'industry' => 'Technology',
+            'skill_level' => 'Advanced',
+        ]);
+        $event = Event::first();
```

Comment:
I looked at the other tests in the codebase and they tend to give a bit more generous line spacing between arrange/act/assert sections, which I like. Can you insert newlines above and below the `$event = Event::first()` line?